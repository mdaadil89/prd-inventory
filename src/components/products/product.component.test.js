import React from 'react'
import {mount, shallow} from 'enzyme'
import Product from './product.component'
import toJson from 'enzyme-to-json';


describe('Products Snapshot', () => {
    let wrapper; 
    let props;
    beforeEach(() => {
        //wrapper = mount(<Product/>);
         
           
            
            wrapper = mount(<Product/>);
    });

    test('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot(); 
    }); 
     
});  