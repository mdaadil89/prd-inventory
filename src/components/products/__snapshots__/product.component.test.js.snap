// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products Snapshot renders correctly 1`] = `
<Product>
  <Row
    noGutters={false}
  >
    <div
      className="row"
    >
       
    </div>
  </Row>
</Product>
`;
